src/components/ChoiceGroup/ChoiceGroup.tsx

```diff
@@ -0,0 +1,112 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import '../../yamui';
+import * as React from 'react';
+import { BaseComponentProps } from '../../util/BaseComponent/props';
+import { ChoiceGroup as FabricChoiceGroup, IChoiceGroupOption } from 'office-ui-fabric-react/lib/ChoiceGroup';
+import { join } from '../../util/classNames';
+import Text from '../Text';
+import './ChoiceGroup.css';
+
+export interface ChoiceGroupOption {
+  /**
+   * A required key to uniquely identify the option.
+   */
+  key: string;
+
+  /**
+   * The label for the option.
+   */
+  label: string;
+
+  /**
+   * The sublabel for the option
+   */
+  sublabel?: string;
+}
+
+export interface ChoiceGroupProps extends BaseComponentProps {
+  /**
+   * Items to be presented to the user.
+   */
+  options: ChoiceGroupOption[];
+
+  /**
+   * Label text that will be rendered above the dropdown.
+   */
+  label?: string;
+
+  /**
+   * Intial selected option. If none is provided and selectionOptional is `false`, the first option is used.
+   */
+  initialSelection?: ChoiceGroupOption['key'];
```

Comment:
i'd suggest replacing `initialSelection` with `selectedKey` and just making this a controlled component, unless you think we need it to be uncontrolled?

Reply:
I think this component should control itself if thats what you mean

Reply:
each app/feature rendering a ChoiceGroup will already be listening to its onChange event and tracking the changed value in its own state, it would just need to pass down a `selectedKey` prop. is there a reason to separately manage the state here and even give the consumer a chance to let the two get out of sync?

Reply:
ah because you care about the final value rather than when it changes. Ok I will think about how to change it

Reply:
we built TextField and TextArea to support both controlled and uncontrolled scenarios, but all instances in yamjs implemented them as controlled by passing down a value prop. since the parent component is already tracking its state via onChange it's not any extra work for the app, it's just a little more work to build the .md example.

looking at Fabric's Choicegroup, i don't think you need to do any state management here to get the uncontrolled scenario for free. i think you can just surface both the defaultSelectedKey and selectedKey and the app can use one or the other